### PR TITLE
*: update doc links

### PR DESCRIPTION
**Pull request by estroz**
**Description of the change:** update website doc links

**Motivation for the change:** broken links

### CODE REVIEW

File: internal/scorecard/scorecard.go [hunk 1]
```diff
@@ -111,9 +112,12 @@ func (s Config) RunTests() error {
...
 }
 
+var lastSDKVersionWithDocDir = semver.MustParse("0.16.0")
+
 func ConfigDocLink() string {
-	if strings.HasSuffix(version.Version, "+git") {
-		return "https://github.com/operator-framework/operator-sdk/blob/master/doc/test-framework/scorecard.md"
+	ver, err := semver.ParseTolerant(version.Version)
+	if strings.HasSuffix(version.Version, "+git") || err != nil || ver.GT(lastSDKVersionWithDocDir) {
+		return "https://sdk.operatorframework.io/docs/scorecard/"
```
Comment (camilamacedo86) [inline internal/scorecard/scorecard.go:120]: it ould be an util centralized. shows that other places could use it.

File: internal/scorecard/scorecard.go [hunk 2]
```diff
@@ -111,9 +112,12 @@ func (s Config) RunTests() error {
...
 	}
 	return fmt.Sprintf(
 		"https://github.com/operator-framework/operator-sdk/blob/%s/doc/test-framework/scorecard.md",
```
Comment (camilamacedo86) [inline internal/scorecard/scorecard.go:123]: should not it point to https://sdk.operatorframework.io/ as well?

File: internal/scorecard/scorecard.go (hunk 1 above)
Comment (estroz) [reply internal/scorecard/scorecard.go:120]: Actually this should always return the SDK website.

File: internal/scorecard/alpha/scorecard.go [hunk 3]
```diff
@@ -165,12 +163,7 @@ func (r FakeTestRunner) RunTest(ctx context.Context, test Test) (result *v1alpha
 }
 
 func ConfigDocLink() string {
-	if strings.HasSuffix(version.Version, "+git") {
-		return "https://github.com/operator-framework/operator-sdk/blob/master/doc/test-framework/scorecard.md"
-	}
-	return fmt.Sprintf(
-		"https://github.com/operator-framework/operator-sdk/blob/%s/doc/test-framework/scorecard.md",
-		version.Version)
+	return "https://sdk.operatorframework.io/docs/scorecard/"
```
Comment (estroz) [inline internal/scorecard/alpha/scorecard.go:166]: @jmccormick2001 are you ok with this change?

Comment (jmccormick2001) [reply internal/scorecard/alpha/scorecard.go:166]: fine with me

**Pull request merged**